engine/api/workflow.go

```diff
@@ -210,9 +208,9 @@ func (api *API) postWorkflowRollbackHandler() service.Handler {
 			return sdk.WrapError(err, "cannot load workflow audit %s/%s", key, workflowName)
 		}
 
-		var exportWf exportentities.Workflow
-		if err := yaml.Unmarshal([]byte(audit.DataBefore), &exportWf); err != nil {
-			return sdk.WrapError(err, "cannot unmarshal data before")
+		exportWf, err := exportentities.UnmarshalWorkflow([]byte(audit.DataBefore))
+		if err != nil {
+			return sdk.WrapError(err, "Cannot unmarshal data before")
```

Comment:
i think errors must not be capitalized
https://github.com/golang/go/wiki/CodeReviewComments#error-strings